### PR TITLE
Removed overriding AI scripts

### PR DESCRIPTION
**Pull request by Freddo3000**

### CODE REVIEW

File: addons/ai/XEH_PREP.hpp [hunk 1]
```diff
@@ -1,9 +1,7 @@
-PREP(assignLoadout);
-PREP(selectItem);
-PREP(selectAttachments);
-PREP(garrison);
-PREP(handleDebugTags);
-PREP(sys_stance);
-PREP(handleLocality);
-PREP(pushLocalCommand);
-PREP(handleWaypointPlaced);
```
Comment (BorderKeeper) [inline addons/ai/XEH_PREP.hpp:9]: This seems to be the shift+click to force AI traversal with careless combat stance. I do not recommend taking that out with the other changes.

File: addons/ai/XEH_postInit.sqf [hunk 2]
```diff
@@ -1,20 +1 @@
...
-
-    [GVAR(stateMachine), {}, {}, {}, "Initial"] call CBA_statemachine_fnc_addState;
-
-    call FUNC(sys_stance);
-};
-
-[QEGVAR(zeus,moduleSetup), {
-    params ["_module"];
-    _module addEventHandler ["CuratorWaypointPlaced", FUNC(handleWaypointPlaced)];
-}] call CBA_fnc_addEventHandler;
```
Comment (BorderKeeper) [inline addons/ai/XEH_postInit.sqf:20]: leave the waypoint module in

File: addons/ai/XEH_unitInit.sqf [hunk 3]
```diff
@@ -1,45 +1,35 @@
...
-
-if (!local _unit) exitWith {};
-
-[{
-    [{
-        params ["_unit"];
-
-        [_unit] call FUNC(assignLoadout);
-
-        _unit disableAI "SUPPRESSION";
```
Comment (BorderKeeper) [inline addons/ai/XEH_unitInit.sqf:17]: I agree with removing this, although I do not know why it was removed exactly in the first place.

File: addons/ai/XEH_unitInit.sqf [hunk 4]
```diff
@@ -1,45 +1,35 @@
...
-        _unit setBehaviour "AWARE";
-        _unit setCombatMode "RED";
-        _unit setSpeedMode "FULL";
-        _unit allowFleeing 0;
```
Comment (BorderKeeper) [inline addons/ai/XEH_unitInit.sqf:21]: Agreed with removing above, although not sure how much of an impact will it make on the problem of AI being too easy to kill. This feels more like a feature to eliminate stragglers from defeated AI squads that Zeus would have to deal with.

**Review by BorderKeeper — CHANGES_REQUESTED** [review]
There are a couple of things that require input/changes from Freddo and couple that need admins input, but the code itself is okay and will work. I hope we can push it into our repo and have this as a foundation for future modifications.

File: addons/ai/XEH_PREP.hpp (hunk 1 above)
Comment (Freddo3000) [reply addons/ai/XEH_PREP.hpp:9]: Yeah it is, and it is simply way too brute force IMO.

```
_unit setUnitPos "UP";
_unit setSpeedMode "FULL";
_unit setBehaviour "AWARE";
_unit setCombatMode "BLUE";

_unit disableAI "TARGET";
_unit disableAI "AUTOTARGET";

_unit allowFleeing 0;

_unit doWatch objNull;
_unit doTarget objNull;
_unit commandWatch objNull;
_unit commandTarget objNull;
```
Especially seeing that there is no notification or indication whenever you're using it.

**Review by RamenNoodleNinja — APPROVED** [review]

**Review by RamenNoodleNinja — APPROVED** [review]

**Review by BorderKeeper — APPROVED** [review]

**Comment by BorderKeeper** [comment]
After a discussion we had on this topic I approve of removal of the waypoint module which was the last thing I was worried about. Merge approved.

**Pull request merged**